app/scripts/components/exploration/components/datasets/colorRangeSlider/color-range-slider.scss

```diff
@@ -48,3 +48,24 @@ input::-webkit-inner-spin-button {
 input[type='number'] {
   -moz-appearance: textfield;
 }
+
+.tooltiptext {
+  background-color: #2c3e50;
+}
+
+.tooltipmin::after {
```

Comment:
@dzole0311 Would you by chance know of a way to leverage pseudo classes in combination with the USWDS util classes?

Reply:
We discussed this question async, and determined that USWDS currently doesn't support pseudo class functionality for this usecase.